priv/repo/migrations/20210406052407_v1_3_1_alter_glific_tables.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Glific.Repo.Migrations.V131AlterGlificTables do
+  use Ecto.Migration
+
+  def change do
+    session_templates()
+  end
+
+  defp session_templates() do
+    alter table(:session_templates) do
+      modify :example, :text, null: true
+    end
```

Comment:
I did not understand this. I think it's text field already, 

Did we make those changes manually?

Reply:
@pankaj-ag  I guess someone made it text field in production but in fresh setup it is varchar with 255 character limit. Similar is case in staging as well